policies/testing.md

```diff
@@ -42,6 +42,46 @@ A test may be omitted where writing the test would result in disproportionately
 more effort than writing the code being tested. For example, difficult to
 reproduce error conditions.
 
+Triage Labels
+-------------
+
+Before approving a PR, the applicability of the testing policy to the PR must
+be assessed and an appropriate label applied to the PR. One of the following
+labels must be applied:
+
+- `tests: present`, to be added where suitable tests are included in the same PR.
+- `hold: tests needed`, where the testing policy requires tests but they are not
```

Comment:
Do some of these tests labels need explicit statements by each approver (I agree with the label X).
(Like we do for trivial)

Reply:
I think it should be up to the approver to disagree with the choice.